parity/test_indexer_v_algod.py

```diff
@@ -0,0 +1,182 @@
+import atexit
+import json
+from pathlib import Path
+from typing import List
+import yaml
+
+from git import Repo
+
+from .json_diff import deep_diff, prettify_diff
+
+NEW, OVERLAP, DROPPED, FULL = "new", "overlap", "dropped", "full"
+DIFF_TYPES = [NEW, OVERLAP, DROPPED, FULL]
+
+# These are the diff reports that will be run and compared/asserted against:
+ASSERTIONS = [DROPPED, FULL]
+
+# Only compare swagger "definitions":
+MODELS_ONLY = True
```

Comment:
@winder - should we consider setting this to `False`? In that case, `responses` would be compares as well.

Reply:
but the counter-argument is that it could get too noisy and harder to read the actual diff (there are more `$ref`'s in responses and they're not all the same in **algod** v **indexer** and these aren't handled especially well by the differ)

Reply:
I think models only is fine. It's mainly the models associated with "Account" that really need to be aligned.